### PR TITLE
Added 3 more translatable strings

### PR DESCRIPTION
**Pull request by DmitriySalnikov**
and fixed russian localization based on user feedback.

Added strings already contained in the current version of the
localization.

### CODE REVIEW

**Comment by DimOkGamer** [comment]
pls merge this first [#4964](https://github.com/godotengine/godot/pull/4964)

**Comment by DimOkGamer** [comment]
oooops.

**Comment by akien-mga** [comment]
> pls merge this first #4964

Done, but there are now conflicts to resolve for this PR.

**Comment by DimOkGamer** [comment]
I'll write when fix it all.

File: tools/translations/ru.po [hunk 1]
```diff
@@ -3215,67 +3215,68 @@ msgstr "Создать контур"
...
 #: tools/editor/plugins/multimesh_editor_plugin.cpp
 msgid "Mesh source is invalid (not a MeshInstance)."
-msgstr "Источник мэша является недействительным (не MeshInstance)."
+msgstr "Источник полисетки является недействительным (не MeshInstance)."
 
 #: tools/editor/plugins/multimesh_editor_plugin.cpp
 msgid "Mesh source is invalid (contains no Mesh resource)."
-msgstr "Источник мэша является недействительным (не содержит ресурс мэша)."
+msgstr ""
+"Источник полисетки является недействительным (не содержит ресурс полисетки)."
```
Comment (DimOkGamer) [inline tools/translations/ru.po]: AaaaaAAAaaAAAaaAA, damn

**Pull request closed without merging**